db/migrate/20200923115939_add_techsource_account_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTechsourceAccountToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :has_techsource_account, :boolean, default: false
```

Comment:
another option is use a time and set this to default to `NULL`

Reply:
I think a boolean here is OK - it's consistent with the `is_support` and `is_computacenter` flags.